app/src/main/res/layout/main_fragment.xml

```diff
@@ -21,7 +21,7 @@
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginBottom="52dp"
-        android:text="Login"
+        android:text="@string/btnLogin_Text"
```

Comment:
changed the hard-coded text "login" in the login button to a value. Not particularly needed, but it's best practice to avoid hard-coded text.